### PR TITLE
add Pharo 11 (make Pharo 10)

### PR DESCRIPTION
**Pull request by estebanlm**

### CODE REVIEW

**Review by fniephaus — CHANGES_REQUESTED** [review]
The builds are failing due to the bash-level tests not being up-to-date. If you think we don't need them, please feel free to drop them.

**Comment by estebanlm** [comment]
no, no, tests are needed!
(I just forgot they were there... doing this once each N months does that... my brain is not the same as before ;) )

**Comment by estebanlm** [comment]
now it should be ready

**Comment by gcotelli** [comment]
We can add Pharo 11 to .github/workflows/main.yml , so at least we will notice if it broke with some future changes.

**Comment by estebanlm** [comment]
yeah, problem is there is no p11 image yet ;)

**Comment by fniephaus** [comment]
If Pharo32-11 is only supported on arm, maybe we shouldn't support it until someone explicitly asked for it and can test it?

Please let me know when there is a Pharo 11 image, so that I can restart the checks and merge this.

**Comment by estebanlm** [comment]
Hi, now we have pharo11 images. 
can you restart the CI and merge if it is appropriate?

**Comment by fniephaus** [comment]
The Pharo 11 build fails with:

> Instance of UnixEnvironment did not understand #getEnv:

https://github.com/hpi-swa/smalltalkCI/runs/6008160625?check_suite_focus=true#step:4:92

**Comment by estebanlm** [comment]
I know :)
working on it.

**Review by gcotelli — APPROVED** [review]
LGTM

**Comment by estebanlm** [comment]
ping

File: repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo11.class/properties.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+		 ],
+	"classvars" : [
+		 ],
+	"commentStamp" : "",
+	"instvars" : [
+		 ],
+	"name" : "SmalltalkCIPharo11",
+	"pools" : [
+		 ],
+	"super" : "SmalltalkCIPharo9",
```
Comment (fniephaus) [inline repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo11.class/properties.json:13]: I think this should eventually inherit from Pharo 10, not 9, right?

**Review by fniephaus — APPROVED** [review]
LGTM, thanks!

**Pull request merged**